src/common/utils.lua

```diff
@@ -196,34 +196,26 @@ function utils.reply(msg)
 	Handlers.utils.reply(msg)
 end
 
-function utils.validateArweaveId(id)
-	local valid = string.match(id, constants.ARWEAVE_ID_REGEXP) == nil
-
-	assert(valid == true, constants.INVALID_ARWEAVE_ID_MESSAGE)
-end
-
-function utils.validateOwner(caller)
-	local isOwner = false
-	if Owner == caller or Balances[caller] or ao.env.Process.Id == caller then
-		isOwner = true
+function utils.parseAntState(data)
+	assert(type(data) == "string", "Data must be a string")
```

Comment:
necessary? does json.decode throw an error if not a valid json string? if so, this can be removed